### PR TITLE
annotateMain

### PR DESCRIPTION
**Pull request by AntonPuko**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -103,11 +113,17 @@ routeConfig(app, [
...
+  response: $Response,
+  next: NextFunction,
+): mixed => response.sendStatus(404);
+
+app.use(noRouteMiddleware);
 
 
 console.log("Starting server, listening on " + NODE_PORT);
-http.createServer(app).listen(NODE_PORT);
+app.listen(NODE_PORT);
```
Comment (AntonPuko) [inline src/main.js]: doesn't make difference wich way to use in our case, but this better for Flow.

File: src/main.js [hunk 2]
```diff
@@ -103,11 +113,17 @@ routeConfig(app, [
   new WebhookController(settings.webhookRepository),
 ]);
 
-app.use((request, response, next) => response.sendStatus(404));
```
Comment (AntonPuko) [inline src/main.js]: do we need this middleware at all in main.js ? or we need to move it to RouteConfig.js somehow?

File: src/main.js [hunk 3]
```diff
@@ -79,10 +89,10 @@ const setCORSHeaders = (request, response, next) => {
     });
     return response.sendStatus(204);
   }
-	else {
-		response.set({'Access-Control-Allow-Origin': '*'});
-		next();
-	}
+  else {
+    response.set({ 'Access-Control-Allow-Origin': '*' });
```
Comment (jlkalberer) [inline src/main.js]: Yeah.. probably.. Or find a different way to set this that conforms to their api.

File: src/main.js (hunk 2 above)
Comment (jlkalberer) [inline src/main.js]: Yeah, I'm thinking that we move all the routing functions to another file. That will call `routeConfig`

**Pull request merged**